spring-cloud-aws-sqs/src/main/java/io/awspring/cloud/sqs/annotation/AbstractListenerAnnotationBeanPostProcessor.java

```diff
@@ -261,14 +261,15 @@ protected void configureDefaultHandlerMethodFactory(DefaultMessageHandlerMethodF
 		CompositeMessageConverter compositeMessageConverter = createCompositeMessageConverter();
 
 		List<HandlerMethodArgumentResolver> methodArgumentResolvers = new ArrayList<>(
-				createAdditionalArgumentResolvers());
+				createAdditionalArgumentResolvers(compositeMessageConverter, this.endpointRegistrar.getObjectMapper()));
 		methodArgumentResolvers.addAll(createArgumentResolvers(compositeMessageConverter));
 		this.endpointRegistrar.getMethodArgumentResolversConsumer().accept(methodArgumentResolvers);
 		handlerMethodFactory.setArgumentResolvers(methodArgumentResolvers);
 		handlerMethodFactory.afterPropertiesSet();
 	}
 
-	protected Collection<HandlerMethodArgumentResolver> createAdditionalArgumentResolvers() {
+	protected Collection<HandlerMethodArgumentResolver> createAdditionalArgumentResolvers(
```

Comment:
Yup, this is a breaking change. We create an overload and have the old method call the new method without parameters. Then we need to make sure we can handle not having the arguments.